pybind11_stubgen/__init__.py

```diff
@@ -300,12 +301,12 @@ class Parser(
     return parser
 
 
-def main():
+def main(args: Sequence[str] | None = None) -> None:
     logging.basicConfig(
         level=logging.INFO,
         format="%(name)s - [%(levelname)7s] %(message)s",
     )
-    args = arg_parser().parse_args(namespace=CLIArgs())
+    args = arg_parser().parse_args(args, namespace=CLIArgs())
```

Comment:
I'm pretty sure mypy would be unhappy with reassingment of args

Reply:
I have renamed it to argv